src/platforms/posix/drivers/gpssim/gpssim.cpp

```diff
@@ -546,8 +578,9 @@ usage(const char *reason)
 	}
 
 	PX4_INFO("usage:");
-	PX4_INFO("gpssim {start|stop|test|reset|status}");
+	PX4_INFO("gpssim {start|stop|test|reset|status|set}");
 	PX4_INFO("       [-d /dev/ttyS0-n][-f (for enabling fake)][-s (to enable sat info)]");
+	PX4_INFO("       [-t # (for setting fix_type)][-n # (for setting # satellites)][-m # (for setting noise multiplier)]");
```

Comment:
Can you extend the description here? What is the default behavior? The distribution and unit for the noise multiplier?

Reply:
done